targets/zephyr/Makefile.zephyr

```diff
@@ -111,6 +131,9 @@ BUILD_CONFIG = O="$(OUTPUT)/zephyr" V=$(V) USER_LIBS="$(LIBS)" USER_LIB_INCLUDE_
 .PHONY: all
 all: jerry zephyr
 
+$(DOTCONFIG):
```

Comment:
As mentioned about $(DOTCONFIG) becomes a dependancy because we include it. If we renamed this build target then the build would break (it would fail when it tried to include the .config file).